src/Http/Controllers/CP/Collections/LocalizeEntryController.php

```diff
@@ -16,7 +16,11 @@ public function __invoke(Request $request, $collection, $entry)
 
         $this->addToStructure($collection, $entry, $localized);
 
-        $localized->store(['user' => User::fromUser($request->user())]);
+        if ($entry->revisionsEnabled()) {
+            $localized->store(['user' => User::fromUser($request->user())]);
+        } else {
+            $localized->published(false)->updateLastModified($request->user())->save();
+        }
```

Comment:
Correct me if I'm wrong but I think this condition can be moved into `Revisable@store`.

```diff
public function store($options = [])
{
    $this
        ->published(false)
        ->updateLastModified($user = $options['user'] ?? false)
        ->save();

+   if ($this->revisionsEnabled()) {
        return $this
            ->makeRevision()
            ->user($user)
            ->message($options['message'] ?? false)
            ->save();
+   }
}
```

Also the return value probably needs to be a little smarter, but for now we can just return the appropriate one - either the entry's or the revision's.

```diff
-$this
+$return = $this
    ->...
    ->save();

if ($this->revisionsEnabled()) {
-    return $this
+    $return = $this
        ->makeRevision()
        ->...
        ->save();
}

+return $return;
```

Reply:
That seems to work, updated!